app/src/main/java/live/hms/app2/ui/meeting/MeetingViewModel.kt

```diff
@@ -197,6 +201,12 @@ class MeetingViewModel(
           }
         }
 
+        override fun onReconnected() {
```

Comment:
Is the SDK version have these callbacks

Reply:
Nope, not yet released (WIP). I'll make these callbacks optional to implement